manual/source/Tutorial/DataStructures/Graphs.rst

```diff
@@ -141,7 +140,10 @@ Assignment 2
 
    Objective
       Write a program which creates a directed graph with the following edges:
-      ``(1,0), (0,4), (2,1), (4,1), (5,1), (6,2), (3,2), (2,3), (7,3), (5,4), (6,5), (5,6), (7,6), (7,7)``
+      ``(1,0), (0,4), (2,1), (4,1), (5,1), (6,2), (3,2), (2,3), (7,3), (5,4),``
+
+      ``(6,5), (5,6), (7,6), (7,7)``
+
```

Comment:
You may want to separate this because of the layout. Right?
I think it will relieve the situation but can't be the solution.
Don't we need to find another way of highlighting for this?